core/optaplanner-constraint-drl/src/main/java/org/optaplanner/constraint/drl/testgen/operation/TestGenKieSessionUpdate.java

```diff
@@ -20,6 +20,7 @@
 import org.kie.api.runtime.KieSession;
 import org.kie.api.runtime.rule.FactHandle;
 import org.optaplanner.constraint.drl.testgen.fact.TestGenFact;
+import org.optaplanner.constraint.drl.testgen.fact.TestGenNullFact;
```

Comment:
How is this possible? CI was green, while if an import was missing, it wouldn't even compile.

Reply:
Javadoc

Reply:
(See ` public TestGenFact getValue()` javadoc for usage)